inbm-lib/inbm_common_lib/utility.py

```diff
@@ -222,7 +222,7 @@ def safe_extract(tarball: tarfile.TarFile,
     tarball.extractall(path, members, numeric_owner=numeric_owner) 
```

Comment:
**Suggestion:** Avoid using 'str.replace' when removing a single character, use 'str.translate' for better performance. [performance]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    return val.translate({ord('\x00'): None})
```